documentation/homework/todo-app/src/component/todos.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import dbTodos from '../todo.json';
+import TodoList from './todoList';
+import TodoForm from './todoForm';
+import Footer from './footer';
+import moment from 'moment';
+
+
+export default class Todos extends React.Component {
+    state = {
+        description: '',
```

Comment:
It took me a minute to realise what you were doing here. Your `description` and `deadline` fields are actually for _new_ todos. It would be better to call them like `newDescription` and `newDeadline`, or put them inside a nested object called `newTodo`.

With confusing names aside, I like that you are keeping all your state in one place!